crm_rma_stock_location/__init__.py

```diff
@@ -19,5 +21,14 @@
 #
 ##############################################################################
 
-from . import stock_warehouse
-from . import product
+from . import models
+from . import wizards
+from openerp import SUPERUSER_ID
+
+
+def post_init_hook(cr, registry):
```

Comment:
@moylop260 yes, as @pedrobaeza says this is a key for a method that executes just after the module is installed.. look over [here](https://github.com/odoo/odoo/blob/8.0/openerp/modules/loading.py#L185) to see when is called.